modules/phase_field/include/postprocessors/FauxGrainTracker.h

```diff
@@ -61,6 +50,7 @@ class FauxGrainTracker : public FeatureFloodCount, public GrainTrackerInterface
   const int _tracking_step;
 
   std::vector<std::pair<unsigned int, unsigned int> > _faux_data;
+  std::vector<unsigned int> _faux_data_2;
```

Comment:
I'd really like to see some doc strings here. The new variable name with the `_2` appended to it is pretty nondescript.